test/pilot.js

```diff
@@ -0,0 +1,34 @@
+/**
```

Comment:
@ngdangtu-vn can we get rid of this file? Doesn't seem to belong into the module?

Reply:
Roger :D